DEVELOPERS.md

```diff
@@ -244,9 +228,19 @@ In that situation you can follow the steps below to set up your local copy of th
    * The other fields don't matter too much for local development.
 1. Register a user account on your local version of job-server by clicking Login
 1. Set the `SOCIAL_AUTH_GITHUB_KEY` (aka "Client ID") and `SOCIAL_AUTH_GITHUB_SECRET` environment variables with values from that OAuth application.
-1. Give your user the `CoreDeveloper` role by:
-   * Setting the `ADMIN_USERS` environment variable to include your username.
-   * Running `python manage.py ensure_admins`.
+1. Give your user the `CoreDeveloper` role by running:
+   ```sh
+   > python3 manage.py shell_plus
+   ```
+
+   and then running in the `shell_plus` shell:
+
+   ```
+   >>> from jobserver.authorization import CoreDeveloper
+   >>> user = User.objects.get(username="<your_username>")
+   >>> user.roles.append(CoreDeveloper)
+   >>> user.save()
```

Comment:
thought: This could reduce to:

```suggestion
   ```sh
   > python3 manage.py make_user_core_developer <your_username>
```

if we go with a management command.

Reply:
Leaving this as I removed the command.